api/files/serializers.py

```diff
@@ -539,13 +539,13 @@ def get_file_download_link(obj, version=None, view_only=None):
 
 
 def get_file_render_link(mfr_url, download_url, version=None):
```

Comment:
Changes to this function are just to effect the insertion order in the `download_url_args` dictionary. Python 3's dictionaries are always ordered by insertion so this is just changed so when urls are generated they always have params in the same order as tests.